static/js/components/SourceDesktop.jsx

```diff
@@ -83,7 +83,7 @@ export const useSourceStyles = makeStyles((theme) => ({
   name: {
     fontSize: "200%",
     fontWeight: "900",
-    color: "darkgray",
+    color: theme.palette.gray[700],
```

Comment:
```suggestion
    color: theme.palette.grey[700],
```
Not sure if this was working for you locally, but I had to change to this to get it to render correctly. Otherwise, got this error:
![Screenshot 2021-02-09 125854](https://user-images.githubusercontent.com/17696889/107407325-8792f480-6ad7-11eb-90e1-d4c42273992d.png)

Reply:
I wasn't seeing errors before, but I'm surprised I wasn't, given `gray` doesn't exist! I'll take a look once this branch builds.